tests/foreman/maintain/test_health.py

```diff
@@ -118,7 +118,12 @@ def test_positive_health_check_by_tags(sat_maintain):
     result = sat_maintain.cli.Health.list_tags().stdout
     output = [i.split("]\x1b[0m")[0] for i in result.split("\x1b[36m[") if i]
     for tag in output:
-        assert sat_maintain.cli.Health.check(options={'tags': tag, 'assumeyes': True}).status == 0
+        assert (
+            sat_maintain.cli.Health.check(
+                options={'tags': tag, 'assumeyes': True, 'whitelist': 'non-rh-packages'}
```

Comment:
can we not remove that non-rh package instead of whitelisting this check here?

Reply:
I don't think we can remove them, we need them for loki(logging)

Reply:
We are installing [Promtail](https://grafana.com/docs/loki/latest/clients/promtail/) on Satellite that sends logs to Loki. And yes, this package is not RH-signed.
So I see two ways to go:
1. whitelist `non-rh-packages`
2. disable Promtail deployment for the ForemanMaintain component

Reply:
@mstrlc @jameerpathan111 Sorry, I haven't seen this PR in long time, so as we install promtail/loki on satellite, I'd have suggested to add this package into plugin whitelist in `/etc/dnf/plugins/foreman-protector.whitelist` and remove it in teardown for f-m tests, just to ensure we're testing this check correctly.